industrial_reconstruction/industrial_reconstruction/industrial_reconstruction.py

```diff
@@ -222,6 +222,7 @@ def startReconstructionCallback(self, req, res):
         self.record = True
 
         res.success = True
+        res.message = "TSDF Volume generated"
```

Comment:
This response is for the start reconstruction call, not the stop, so it doesn't have a message field.